### ISSUE TITLE
Selftrained Model does nothing

### ISSUE THREAD

**Issue by Mar-Pfa** [deezer/spleeter#623]
i'm currently trying to train a model but i'm not successfull 
(same problem as e.g. here: https://www.reddit.com/r/spleeter/comments/jtvlg5/self_trained_model_is_doing_nothing/)

i've feeding the training with several Hours of Material, 2Stems, 
- after separation the both outpufiles are completely the same instead of anything happens
- the training itself runs very fast on ~20GB Wave data - i through below a minute - looks like it does not really train but i get no error message.... is there a other log where i can check the training success?
- or do i have to run the training multiple times?

**Comment by romi1502** [comment]
Hi @Mar-Pfa
There is definitely something strange happening with your training. 
Training a model from scratch should take (at least) hours.
What config file do you use for training?
Also, have you tried to run training with the `--verbose` option to check what may be going wrong?

**Comment by Mar-Pfa** [comment]
thanks! i was not aware of the --verbose option :-/

Spleeter Version: 2.2.2 running on Windows / Conda

i tried several things but could not find a solution so far :-( here is my output and base config:

**C:\repo\spleeter\spleeter> spleeter train -d C:\repo\spleetertest\tst -p C:\repo\spleetertest\tst\base_config.json --verbose**

INFO:tensorflow:Using config: {'_model_dir': 'C:\\repo\\spleetertest\\tst\\model', '_tf_random_seed': 0, '_save_summary_steps': 5, '_save_checkpoints_steps': 150, '_save_checkpoints_secs': None, '_session_config': gpu_options {
  per_process_gpu_memory_fraction: 0.45
}
, '_keep_checkpoint_max': 2, '_keep_checkpoint_every_n_hours': 10000, '_log_step_count_steps': 10, '_train_distribute': None, '_device_fn': None, '_protocol': None, '_eval_distribute': None, '_experimental_distribute': None, '_experimental_max_worker_delay_secs': None, '_session_creation_timeout_secs': 7200, '_service': None, '_cluster_spec': ClusterSpec({}), '_task_type': 'worker', '_task_id': 0, '_global_id_in_cluster': 0, '_master': '', '_evaluation_master': '', '_is_chief': True, '_num_ps_replicas': 0, '_num_worker_replicas': 1}
INFO:spleeter:Start model training
INFO:tensorflow:Not using Distribute Coordinator.
INFO:tensorflow:Running training and evaluation locally (non-distributed).
INFO:tensorflow:Start train and evaluate loop. The evaluate will happen after every checkpoint. Checkpoint frequency is determined based on RunConfig arguments: save_checkpoints_steps 150 or save_checkpoints_secs None.
WARNING:tensorflow:From c:\programdata\anaconda3\lib\site-packages\tensorflow\python\training\training_util.py:235: Variable.initialized_value (from tensorflow.python.ops.variables) is deprecated and will be removed in a future version.
Instructions for updating:
Use Variable.read_value. Variables in 2.X are initialized automatically both in eager and graph (inside tf.defun) contexts.
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Apply unet for vocals_spectrogram
INFO:tensorflow:Apply unet for accompaniment_spectrogram
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Create CheckpointSaverHook.
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from C:\repo\spleetertest\tst\model\model.ckpt-0
WARNING:tensorflow:From c:\programdata\anaconda3\lib\site-packages\tensorflow\python\training\saver.py:1077: get_checkpoint_mtimes (from tensorflow.python.training.checkpoint_management) is deprecated and will be removed in a future version.
Instructions for updating:
Use standard file utilities to get mtimes.
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
INFO:tensorflow:Calling checkpoint listeners before saving checkpoint 0...
INFO:tensorflow:Saving checkpoints for 0 into C:\repo\spleetertest\tst\model\model.ckpt.
INFO:tensorflow:Calling checkpoint listeners after saving checkpoint 0...
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Apply unet for vocals_spectrogram
INFO:tensorflow:Apply unet for accompaniment_spectrogram
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Starting evaluation at 2021-05-21T05:06:04Z
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from C:\repo\spleetertest\tst\model\model.ckpt-0
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
INFO:tensorflow:Inference Time : 1.80602s
INFO:tensorflow:Finished evaluation at 2021-05-21-05:06:06
INFO:tensorflow:Saving dict for global step 0: absolute_difference = 1.9979906, accompaniment_spectrogram = 0.9741285, global_step = 0, loss = 1.9979906, vocals_spectrogram = 1.0238621
INFO:tensorflow:Saving 'checkpoint_path' summary for global step 0: C:\repo\spleetertest\tst\model\model.ckpt-0
WARNING:tensorflow:Training with estimator made no steps. Perhaps input is empty or misspecified.
INFO:tensorflow:Loss for final step: None.
INFO:spleeter:Model training done

**type C:\repo\spleetertest\tst\base_config.json**

{
        "train_csv": "C:\\repo\\spleetertest\\tst\\train.csv",
        "validation_csv": "C:\\repo\\spleetertest\\tst\\validation.csv",
        "model_dir": "C:\\repo\\spleetertest\\tst\\model",
        "mix_name": "mix",
        "instrument_list": ["vocals", "accompaniment"],
        "sample_rate":44100,
        "frame_length":4096,
        "frame_step":1024,
        "T":512,
        "F":1024,
        "n_channels":2,
        "n_chunks_per_song":40,
        "separation_exponent":2,
        "mask_extension":"zeros",
        "learning_rate": 1e-4,
        "batch_size":4,
        "training_cache":"C:\\repo\\spleetertest\\tst\\training_cache",
        "validation_cache":"C:\\repo\\spleetertest\\tst\\validation_cache",
        "train_max_steps": 1000000,
        "throttle_secs":300,
        "random_seed":0,
        "save_checkpoints_steps":150,
        "save_summary_steps":5,
        "model":{
                        "type":"unet.unet",
                        "params":{
               "conv_activation":"ELU",
               "deconv_activation":"ELU"
                        }
        }
}

**type C:\repo\spleetertest\tst\train.csv**

mix_path,vocals_path,accompaniment_path,duration
C:\repo\spleetertest\tst\output0.wav,C:\repo\spleetertest\tst\output1.wav,C:\repo\spleetertest\tst\output2.wav,200.0

**dir C:\repo\spleetertest\tst\output*.wav**

Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
-a----        12.05.2021     23:51       35340366 output0.wav
-a----        12.05.2021     23:51       35340366 output1.wav
-a----        12.05.2021     23:51       35340366 output2.wav

**Comment by Mar-Pfa** [comment]
Crazy -> found it:

changed the base config from

	"training_cache":"C:\\repo\\spleetertest\\tst\\training_cache",
	"validation_cache":"C:\\repo\\spleetertest\\tst\\validation_cache",

to

	"training_cache":"C:\\repo\\spleetertest\\tst\\training_cache\\",
	"validation_cache":"C:\\repo\\spleetertest\\tst\\validation_cache\\",

and now it works

**Comment by Mar-Pfa** [comment]
and another thing i've learned:
on windows the csv with the training files should not contain absolut folders e.g. like 
C:\train\file1.mp3
but only relative folders (e.g. file1.mp3) - then you have to ensure the position... i think the audio-files are searched based on the config.json file location!? 
at least: i place my training audio files now in the config.json directory and training works :-)

**Comment by MohammedMehdiTBER** [comment]
Same problem here. I kept it training for hours and It didn't stop. 
Here is my config:


{
    "train_csv": "configs/musdb_train.csv",
    "validation_csv": "configs/musdb_validation.csv",
    "model_dir": "musdb_model_16Khz",
    "mix_name": "mix",
    "instrument_list": ["vocals", "drums", "bass", "other"],
    "sample_rate":48000,
    "frame_length":4096,
    "frame_step":1024,
    "T":512,
    "F":1536,
    "n_channels":2,
    "n_chunks_per_song":40,
    "separation_exponent":2,
    "MWF":1,
    "mask_extension":"zeros",
    "learning_rate": 1e-4,
    "batch_size":4,
    "training_cache":"cache/training",
    "validation_cache":"cache/validation",
    "train_max_steps": 200000,
    "throttle_secs":1800,
    "random_seed":3,
    "save_checkpoints_steps":1000,
    "save_summary_steps":5,
    "model":{
        "type":"unet.unet",
        "params":{
               "conv_activation":"ELU",
               "deconv_activation":"ELU"
        }
    }
}

And here is the process folder 
![image](https://user-images.githubusercontent.com/78225592/167283066-61a5dd09-02a4-433a-ad67-f2dcc8418e72.png)
